tools/jhls/Makefile.sub

```diff
@@ -9,9 +9,13 @@ jhls_LIBS = \
 	libtooling \
 	libhls \
 	libllvm \
+	libmlir \
 	librvsdg \
 	libutil \
 
-jhls_EXTRA_LDFLAGS = $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) \
+jhls_EXTRA_LDFLAGS = \
+	$(shell $(LLVMCONFIG) --libs core irReader) \
+	$(shell $(LLVMCONFIG) --ldflags) \
+	${MLIR_LDFLAGS} \
```

Comment:
The same here. Why is there all in a sudden a dependency on libmlir here?

Reply:
This has been discussed earlier and is item 3 in issue https://github.com/phate/jlm/issues/379